csharp/ql/lib/semmle/code/csharp/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -750,6 +750,27 @@ module Private {
       )
     }
 
+    /**
+     * Holds if `p` can reach `n` in a summarized callable, using only value-preserving
+     * local steps. `clearsOrExcepts` records whether any node on the path from `p` to
```

Comment:
`s/clearsOrExcepts/clearsOrExpects` everywhere, right?
Otherwise looks plausible to me.